articles/Setup-Windows-Python.md

```diff
@@ -50,7 +55,26 @@ setx PATH "C:\local\opencv3.10\build\x64\vc14\bin;%PATH%"
 
 This page will walk you through the process of installing the Microsoft Cognitive Toolkit (CNTK) to use from Python in Windows. If you are looking for any other kind of support to setup a CNTK build environment or installing CNTK on your system, you should go [here](./Setup-CNTK-on-your-machine.md) instead. 
 
-Depending on the Python and CNTK version (CPU or GPU) we supply different wheel (.whl) files to install CNTK. Please select the correct installation from the list below, and substitute the name and/or link during the installation.
+We offer three ways to install CNTK for Python:
+1. [PyPI installation](#1-install-from-pypi)
+2. [Wheel (.whl) files for each release](#2-install-from-wheel-files)
+3. [Nightly builds](#3-install-from-nightly-builds)
+
+### 1. Install from PyPI
```

Comment:
Should we reinforce the requirements (opencv, mkl and/or cuda) before teaching the pip install command ?

We should also let clear that the user should not have cntk and cntk-gpu installed simultaneously. One might overwrite the other

We should also advise to pip uninstall previous versions of cntk/cntk-gpu before installing a new one